calcs/strain.py

```diff
@@ -131,7 +131,7 @@ def h_c_n(m_c, f_orb, ecc, n, dist, interpolated_g=None):
         # unsort the output array if there is more than one eccentricity
         if isinstance(ecc, (np.ndarray, list)) and len(ecc) > 1:
             g_vals = g_vals[np.argsort(ecc).argsort()]
-        n_dependent_part = (g_vals / N)**(0.5) / N
```

Comment:
And this was the strain bug, for some reason I chose to put an `N` inside _and_ outside of the parentheses...? Foolish younger me